vl-grid.src.js

```diff
@@ -44,14 +44,18 @@ export class VlLayout extends NativeVlElement(HTMLDivElement) {
  * De grid(.vl-grid) dient om de lay-out van jouw pagina in orde te brengen.
  * Je kan vl-grid vergelijken met de Row element in Bootstrap.
  * @extends VlElement
- * 
+ *
  * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-grid/releases/latest|Release notes}
  * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-grid/issues|Issues}
  * @see {@link https://webcomponenten.omgeving.vlaanderen.be/demo/vl-grid.html|Demo}
  */
 export class VlGrid extends NativeVlElement(HTMLDivElement) {
   static get _observedClassAttributes() {
-    return ['is-stacked']
+    return [
+      'is-stacked',
+      'align-start', 'align-center', 'align-end',
+      'align-space-between', 'align-space-around'
+    ];
```

Comment:
Documentatie ontbreekt nog, maar zal door @SanderKleykens voorzien worden in een volgende PR.